source/sass/glyphs.scss

```diff
@@ -1,26 +1,43 @@
-@mixin glyph($id, $width: 20px, $height: 20px) {
+@mixin glyph($id, $buyers-guide: false, $width: 20px, $height: 20px) {
```

Comment:
Same. 

Instead of having the 2nd parameter being a `boolean`, let's make it a string called `path-to-directory` with default value `../_images/glyphs/`. This way more projects/apps on Foundation site will be able to utilize these mixins better.